### PR TITLE
Add download page

### PR DESCRIPTION
**Pull request by afshin**
This PR adds a download page for downloading mini-forge installers.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
I think this is clearly a net plus compared to the current state of things. I would be +1 for this to be merged.

**Comment by afshin** [comment]
I tried this on mobile, I'll fix the two buttons cutting each other off tomorrow.

image

File: src/components/Download/index.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <div className={styles.header_content}>
+                <h1>
+                    Download an installer
+                </h1>
+                <div>
+                    <p>Latest installers with Python 3.10 <code>(*)</code> in the base environment:</p>
+                    {downloads.map(({ os, arch, icon, href }, index) => (
+                        <Link key={index} href={href}>
+                            <button><i className={icon}></i>{` ${os} – ${arch}`}</button>
+                        </Link>
```
Comment (jaimergp) [inline src/components/Download/index.jsx]: This creates a strange interaction on hover and click:

- My cursor doesn't change when I hover the button
- Instead, it changes outside of the button, by the right side 🤔 
- The whole area is linked though, so if you click around it, it gets you there.
- There's no hover or active status for the link.

Maybe using the regular `<a>` tag plus some classes would work. Or a `Card`?

Comment (jaimergp) [reply src/components/Download/index.jsx]: Another option would be to copy the tiles of the frontpage (the Supporters grid).

File: src/components/Header/index.jsx [hunk 2]
```diff
@@ -30,6 +30,13 @@ export default function Header() {
                     >
                         Explore conda-forge
                     </Link>
+                    <p>&nbsp;</p>
+                    <Link
+                        to="/download"
+                        className="button button--secondary button--lg homepage_button"
+                    >
+                        Download conda-forge
```
Comment (jaimergp) [inline src/components/Header/index.jsx]:
```suggestion
                        Download installer

```

File: src/components/Download/index.jsx (hunk 1 above)
Comment (SylvainCorlay) [reply src/components/Download/index.jsx]: The supporters grid looks nice. This may be a good option.

Comment (trallard) [reply src/components/Download/index.jsx]: > This creates a strange interaction on hover and click:

The `button` component is technically within the  `<a>` (see screenshot), messing with the interactive states and cursor/pointer.
 
🚧_cf-infra-docs_🚧___🚧_cf-infra-docs_🚧

> Maybe using the regular <a> tag plus some classes would work. Or a Card?

Considering the use and purpose of this (i.e., connecting to another resource & downloading the linked resource), it is usually recommended to use an anchored element/link [^1]. So I'd suggest replacing it with a link (can pre-prend the OS icon), or if we want a fancier display, the cards/grid approach would be good too. 

[^1]: [Should I use a button or a link?](https://ashleemboyer.com/blog/should-i-use-a-button-or-a-link#skip-navigation-link)

**Comment by afshin** [comment]
pre-commit.ci autofix

**Comment by SylvainCorlay** [comment]
I think this is good to go as a first version (I expect an insane amount of bikeshedding over this page, so let's get something in, and iterate).

**Comment by jaimergp** [comment]
Agreed. Let's submit this to https://github.com/conda-forge/conda-forge.github.io and see what happens. For example, I think it would be nice to have the sha256 hashes published, but that will require some changes in the miniforge repo or some API calls, idk yet.

**Comment by afshin** [comment]
This PR is superseded by: https://github.com/conda-forge/conda-forge.github.io/pull/2116

**Pull request closed without merging**